### ISSUE TITLE
[TF 2.0 API Docs] tf.keras.layers.GRU

### ISSUE THREAD

**Issue by math-alpha** [tensorflow/tensorflow#29860]
## URL(s) with the issue:

https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/layers/GRU#properties

### Clear description

Initialising float variables using 0. rather than 0.0

dropout=0.,
recurrent_dropout=0.,

### Parameters defined
   
 time_major Argument not documented.

### Raises listed and defined
No

### Usage example
No

**Comment by ymodak** [comment]
@math-alpha Can you please elaborate why you want to make that change? I think current initialization is correct.
```python
dropout=0.0,
recurrent_dropout=0.0
```

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by math-alpha** [comment]
ok
